apps/balancer-tools/src/app/poolsimulator/(utils)/index.ts

```diff
@@ -63,13 +63,57 @@ export function convertAnalysisDataToAMM(data: AnalysisData) {
             dSq: String(data.poolParams?.dSq),
           },
           tokenRates: data.tokens.map((token) => String(token.rate)),
-        }),
+        })
       );
     }
+    // case PoolTypeEnum.Gyro2: {
+    //   console.log({
```

Comment:
I assume that you add this to be used in another task, right? Could you refer to the issue here?

Reply:
yup, the math tasks. Will add here